tools/flaky-test-reporter/config/config.go

```diff
@@ -39,8 +39,9 @@ type Config struct {
 
 // JobConfig is initial configuration for a given repo, defines which job to scan
 type JobConfig struct {
-	Name          string         `yaml:"name"` // name of job to analyze
-	Repo          string         `yaml:"repo"` // repository to test job on
+	Name          string         `yaml:"name"`                  // name of job to analyze
+	Org           string         `yaml:"org" default:"knative"` // org to test job on
```

Comment:
I'd prefer to be explicit, this will move along the lines of making this tool work for any Github repos

Reply:
I agree. Done.